Libplanet.Explorer/Store/AddressRefDoc.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Libplanet.Tx;
+using LiteDB;
+
+namespace Libplanet.Explorer.Store
+{
+    public class AddressRefDoc
```

Comment:
Why is this public while `TxRefDoc` is internal?

Reply:
It seems mistake. It should be also internal.